.github/workflows/schema.yml

```
@@ -0,0 +1,37 @@
+name: Update JSON schema
+
+on: #workflow_dispatch
+  push
+
+    #paths:
+    #- '../../schemas/schema.json'
```

Update the schema under `jina-ai/api` only when `schema.json` is changed.
```suggestion
on:
  push
    branches:
    - master
    paths:
    - '../../schemas/schema.json'
```